### PR TITLE
Add new information about the coin set model.

### PR DESCRIPTION
**Pull request by danieljperry**

### CODE REVIEW

File: docs/04coin-set-model/coin_set_vs_account.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+Each user typically has just one account. It's possible to create additional accounts in order to use more than one address. However, high fees typically discourage users from doing this. Also, it's easy to blacklist an ETH address.
+
+### Determinism
+
+#### Coin set
+
+Coins can only be spent once, making the results deterministic (if you run the same program multiple times, you'll get the same result each time). Because of this, re-applying mempool transactions after a new block is not necessary.
+
+Note, however, that smart coins _could_ be designed such that multiple people could spent the same coin, which would potentially reduce determinism for those coins.
```
Comment (hoffmang9) [inline docs/04coin-set-model/coin_set_vs_account.md:92]: Should be `could spend`

File: docs/04coin-set-model/coin_set_vs_account.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 7
+---
+
+# 4.7 Coin Set vs Account
+
+In the account model used in many blockchains such as Ethereum, Solana and EOS, balances are kept in accounts. These are permanent data structures which do not get destroyed when they send funds.
```
Comment (hoffmang9) [inline docs/04coin-set-model/coin_set_vs_account.md:7]: Lets replace EOS with Algorand

**Review by hoffmang9 — CHANGES_REQUESTED** [review]
Looks good with two nits.

**Comment by danieljperry** [comment]
Nits updated.

**Review by hoffmang9 — APPROVED** [review]
lgtm

**Pull request merged**